examples/Demo.Console.NETFramework472/Program.cs

```diff
@@ -39,14 +39,10 @@ internal class Program
         public string Tenant { get; set; }
 
         /// <summary>
-        /// The on execute.
+        /// This method executes the auth process. This method is invoked when lasso is launched.
         /// </summary>
-        /// <param name="logger">
-        /// The logger.
-        /// </param>
-        /// <returns>
-        /// The <see cref="int"/>.
-        /// </returns>
+        /// <param name="logger"> The logger. </param>

```

Comment:
No need for the extra spaces, I think.

```suggestion
        /// <param name="logger">The logger.</param>
```